fineract-avro-schemas/src/main/avro/loan/v1/LoanAccountDelinquencyRangeDataV1.avsc

```diff
@@ -34,6 +34,33 @@
                 "null",
                 "org.apache.fineract.avro.loan.v1.DelinquencyRangeDataV1"
             ]
+        },
+        {
+            "default": null,
+            "name": "charges",
+            "type": [
+                "null",
+                {
+                    "type": "array",
+                    "items": "org.apache.fineract.avro.loan.v1.LoanChargeDataRangeViewV1"
+                }
+            ]
+        },
+        {
+            "default": null,
+            "name": "currency",
+            "type": [
+                "null",
+                "org.apache.fineract.avro.generic.v1.CurrencyDataV1"
+            ]
+        },
+        {
+            "default": null,
+            "name": "amount",
```

Comment:
Can we add some description to this to make it more self-explanatory?

Reply:
description added